### PR TITLE
How to create a storage pool

### PR DESCRIPTION
**Pull request by ru-fu**
Add general information on how to create a storage pool and move the examples from the reference docs to the how to. 
Also clean up and align the example commands, and add commands for CephFS.

### CODE REVIEW

File: doc/howto/storage_create_pool.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+```
+```{group-tab} Btrfs
+
+Create a loop-backed pool named `pool1`:
+
+     lxc storage create pool1 btrfs
+
+Use the existing Btrfs file system at `/some/path` for `pool2`:
+
+     lxc storage create pool2 btrfs source=/some/path
```
[the remark this answers is not in the document]
Comment (stgraber) [reply doc/howto/storage_create_pool.md:37]: Yep, all storage pools should have it.

File: doc/howto/storage_create_pool.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+     lxc storage create pool2 ceph ceph.cluster_name=my-cluster
+
+Create an OSD storage pool named `pool3` with the on-disk name `my-osd` in the default Ceph cluster:
+
+     lxc storage create pool3 ceph ceph.osd.pool_name=my-osd
+
+Use the existing OSD storage pool `my-already-existing-osd` for `pool4`:
+
+     lxc storage create pool4 ceph source=my-already-existing-osd
```
[the remark this answers is not in the document]
Comment (stgraber) [reply doc/howto/storage_create_pool.md:107]: Yep

File: doc/howto/storage_create_pool.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+Use the existing OSD erasure-coded pool `ecpool` and the OSD replicated pool `rpl-pool` for `pool5`:
+
+     lxc storage create pool5 ceph source=rpl-pool ceph.osd.data_pool_name=ecpool
+```
+```{group-tab} CephFS
```
[the remark this answers is not in the document]
Comment (stgraber) [reply doc/howto/storage_create_pool.md:113]: Looks good, you may want to add:
```
lxc storage create poolX cephfs source=my-filesystem/sub-directory
```

**Review by tomponline — APPROVED** [review]
LGTM

**Comment by tomponline** [comment]
@ru-fu out of interest, is the lack of upper case "d" for the "dir" item on this page https://linuxcontainers.org/lxd/docs/master/reference/storage_drivers/ intentional?

image

**Comment by tomponline** [comment]
> > @ru-fu out of interest, is the lack of upper case "d" for the "dir" item on this page https://linuxcontainers.org/lxd/docs/master/reference/storage_drivers/ intentional? I think it looks rather jarring.
> > image
> 
> No, it isn't - it should probably say "Directory" here. And we'll need to make clear (on all driver pages) how to specify the driver (with `dir`).

Great thanks, we need something like that for all storage drivers as they are case sensitive in the `lxc storage create` command.

**Pull request merged**